views/transportation/tabs.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+// import Icon from 'react-native-vector-icons/Entypo'
+
+import OtherModesView from './otherModes'
+import BusView from './bus'
+// import PauseMenuView from './pause'
+
+export default [
+  {
+    id: 'bus',
+    title: 'Bus',
+    rnVectorIcon: {
+      iconName: 'bus',
+      selectedIconName: 'bus',
```

Comment:
For future reference: these are the icons you can choose from. http://www.entypo.com/

`bus` is not one of them 😝